src/steps/finish/upload.tsx

```diff
@@ -184,9 +184,16 @@ const UploadForm: React.FC<UploadFormProps> = ({ handleUpload }) => {
     }
   }
 
-  // If the user has not yet changed the value of the field and the last used
-  // presenter name is used in local storage, use that.
+  // If presenter field is empty use the user.name from `/info/me.json` otherwise
+  // use value from local storage
```

Comment:
```suggestion
  // If presenter field is empty, use the user.name from `/info/me.json`. Otherwise
  // use value from local storage.
```

But apart from that comma usage, that's an interesting question: is the info from `info/me.json` really more important than the one in local storage? If someone wants a different presenter name than the `info/me.json` name, then having to adjust that every time is annoying I suppose. So I would argue for prioritizing local storage.